data/variable/database/templates/users.mysql.sql

```diff
@@ -2,7 +2,7 @@ CREATE TABLE users (
 
     id       INTEGER UNSIGNED NOT NULL PRIMARY KEY AUTO_INCREMENT,
     username VARCHAR(50),
-    digesta1 VARCHAR(32),
+    digesta1 VARCHAR(60),
```

Comment:
Both digesta1 and username should probably be a `VARBINARY` instead. I'm gonna change that in sabre/dav now as well.